validator/main.go

```diff
@@ -591,14 +616,26 @@ func runCommandWithWait(command string, args []string, sleepSeconds int, silent
 	}
 }
 
-func getDriverRoot() (string, bool) {
-	// check if driver is pre-installed on the host and use host path for validation
-	if fileInfo, err := os.Lstat("/host/usr/bin/nvidia-smi"); err == nil && fileInfo.Size() != 0 {
-		log.Infof("Detected pre-installed driver on the host")
-		return "/host", true
+// prependPathListEnvvar prepends a specified list of strings to a specified envvar and returns its value.
+func prependPathListEnvvar(envvar string, prepend ...string) string {
+	if len(prepend) == 0 {
+		return os.Getenv(envvar)
 	}
+	current := filepath.SplitList(os.Getenv(envvar))
+	return strings.Join(append(prepend, current...), string(filepath.ListSeparator))
+}
 
-	return driverContainerRoot, false
+// setOrOverrideEnvvar adds or updates an envar to the list of specified envvars and returns it.
+func setOrOverrideEnvvar(envvars []string, key, value string) []string {
```

Comment:
I think `set` more or less implies that it will override the value if it has already been set previously
```suggestion
func setEnvVar(envvars []string, key, value string) []string {
```

Reply:
Done